volto/custom-views.rst

```diff
@@ -0,0 +1,8 @@
+.. _custom_views-label:
+
+============
+Custom Views
+============
+
+
+add custom view => move summary view to all content view
```

Comment:
This makes no sense to me. Is this just a WIP placeholder?

Reply:
it is.